### PR TITLE
Added keywords: libname set output options

### PR DESCRIPTION
**Pull request by zaborek**

### CODE REVIEW

**Comment by RunDevelopment** [comment]
Thank you for your contribution!

However, you seem to have forgotten to commit the language file `prism-sas.js`.

Also, please add the new keywords to the SAS [keyword test](https://github.com/PrismJS/prism/blob/master/tests/languages/sas/keyword_feature.test). More infos on tests can be found [here](https://prismjs.com/test-suite.html#writing-tests).

File: tests/languages/sas/keyword_feature.test [hunk 1]
```diff
@@ -1,13 +1,15 @@
 data else format if
 input proc run then
+libname set output options
 
 ----------------------------------------------------
 
 [
 	["keyword", "data"], ["keyword", "else"], ["keyword", "format"], ["keyword", "if"],
-	["keyword", "input"], ["keyword", "proc run"], ["keyword", "then"]
+	["keyword", "input"], ["keyword", "proc run"], ["keyword", "then"], ["keyword", "libname],
```
Comment (RunDevelopment) [inline tests/languages/sas/keyword_feature.test]: You missed a quote `"` there after `libname`. That's why the test is failing.

Comment (zaborek) [reply tests/languages/sas/keyword_feature.test]: Thanks for bearing with me; this is my first github edit.

**Comment by RunDevelopment** [comment]
Thank you for contributing!

**Pull request merged**